### ISSUE TITLE
Ga.preset does not work

### ISSUE THREAD

**Issue by eric-wieser** [pygae/galgebra#217]
```
>>> from galgebra.ga import Ga
>>> Ga.preset('o3d')
AttributeError: type object 'Ga' has no attribute 'set_simp'
```
Tempting to just remove this

**Comment by utensil** [comment]
The preset functionality should not be removed or at least should be implemented in a new way. It's a shorthand to create frequently used GAs. We can deduct the behaviour of `set_simp` is actually just to set a few attributes such as `norm=True`.

**Comment by utensil** [comment]
We should provide a new Ga.preset implementation in the coming release, covering frequently used GAs with canonical symbols for coordinates and this is a common feature in GA libraries.

x-ref: #395

**Comment by eric-wieser** [comment]
I'd probably lean towards a `galgebra.presets` module or subpackage.

**Comment by utensil** [comment]
> I'd probably lean towards a `galgebra.presets` module or subpackage.

Agreed. That's a newly designed API and as it contains a rich set of possibilities and customization code, particularly there could a lot of new methods dedicated for PGA and CGA.
